### DISCUSSION TITLE
Nested filter [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5862] [Discord Threads]
> 🙋‍♂️ **jangxx**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
####   
![image0](https://cdn.discordapp.com/attachments/1101475863796650054/1101521539364560896/Screenshot_2023-04-28_at_8.22.28_PM.png)  
#  
> 🙋‍♂️ **jangxx**  
#### okay thanks 👍  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1101475863796650054)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
